filament/src/details/View.h

```diff
@@ -217,6 +217,8 @@ class FView : public View {
     }
 
     void setRenderTarget(FRenderTarget* renderTarget) noexcept {
+        assert_invariant(!mMultiSampleAntiAliasingOptions.enabled ||
+                !renderTarget->hasSampleableDepth());
```

Comment:
I think we could be setting a `nullptr` renderTarget here (e.g. to "unset" it).

Reply:
Excellent catch, fixed.